tests/mock_data/items.py

```diff
@@ -0,0 +1,20 @@
+json = [
```

Comment:
Slightly confusing for this to be called `json` when it's just python code, not json.